swaybar/input.c

```diff
@@ -207,7 +207,7 @@ static void workspace_next(struct swaybar *bar, struct swaybar_output *output,
 		}
 	}
 
-	if (new) {
+	if (new && new != active) {
```

Comment:
We could also just swap the order of the assignments within this block, but maybe that's too subtle. Up to you.

Reply:
Yeah there were various ways to do it but I figured it was best to avoid a pointless ipc command if we can